shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/DDLStatement.g4

```diff
@@ -998,3 +998,91 @@ supplementalLogGrpClause
 supplementalIdKeyClause
     : DATA LP_ (ALL | PRIMARY KEY | UNIQUE | FOREIGN KEY) (COMMA (ALL | PRIMARY KEY | UNIQUE | FOREIGN KEY))* RP_ COLUMNS
     ;
+
+alterSession
+    : ALTER SESSION alterSessionOption
+    ;
+
+alterSessionOption
+    : adviseClause
+    | closeDatabaseLinkClause
+    | commitInProcedureClause
+    | securiyClause
+    | parallelExecutionClause
+    | resumableClause
+    | shardDdlClause
+    | SYNC WITH PRIMARY
```

Comment:
Why not put `SYNC WITH PRIMARY` in a rule?

Reply:
Yes, we can put it in a rule. I'll put it in a rule like: 
```
syncWithPrimaryClause: SYNC WITH PRIMARY
```

Reply:
Good. After you finished, tell me, please.

Reply:
I've finished it @wgy8283335